gettsim/social_insurance_contributions/ges_krankenv.py

```diff
@@ -34,31 +31,127 @@ def ges_krankenv_beitr_m(
     -------
 
     """
-    beitr_regulär_beschäftigt_m = (
-        soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]["an"]
-        * _ges_krankenv_beitr_bruttolohn_m
-    )
 
     if selbstständig:
         out = ges_krankenv_beitr_selbst_m
     elif geringfügig_beschäftigt:
         out = 0.0
     elif in_gleitzone:
-        out = _ges_krankenv_beitr_midi_job_m
+        out = _ges_krankenv_midi_job_arbeitn_m
     else:
-        out = beitr_regulär_beschäftigt_m
+        out = _ges_krankenv_beitr_reg_beschäftigt
 
     # Add the health insurance contribution for pensions
     return out + ges_krankenv_beitr_rente_m
 
 
-def _ges_krankenv_beitr_bruttolohn_m(
+def ges_krankenv_beitr_satz_bis_2018(soz_vers_beitr_params: dict) -> float:
+    """Select contribution rates of employees for health insurance until 2018.
+
+    The contribution rates consists of a general rate (split equally between employers
+    and employees) and a top-up rate which is fully paid by employees.
+
+    Parameters
+    ----------
+    soz_vers_beitr_params
+        See params documentation :ref:`soz_vers_beitr_params <soz_vers_beitr_params>`.
+
+    Returns
+    -------
+
+    """
+    beitr_satz_params = soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]
```

Comment:
```suggestion
    params = soz_vers_beitr_params["beitr_satz"]["ges_krankenv"]
```
This is just a short function, so fine have a shorthand to remove context. 

The prior version is actually confusing without seeing this line because it is the same as the outer dict without the `ges_krankenv` specifier.